Tests/Tests/Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Tests.Tests
+{
+    public static class Constants
+    {
+        public const string TestEthereumAddress = "0x646934251e9045F41DdF7145Dea5cD63cD45d8Da";
```

Comment:
This whole class can contain all 0x related addresses (on the popular Ethereum blockchains that we use).

Reply:
Yes